src/Table/docs/index.story.js

```diff
@@ -148,10 +149,16 @@ export default {
               title: 'Table with Toolbar',
               description:
                 'This is a basic table that implements a toolbar using `<Table.ToolbarContainer/>` - which receives the SelectionContext (`showSelection` is true in order to enable the selection). Notice that the whole table is wrapped by `<Card/>` so that the toolbar seems like part of the table. In addition, we render the main toolbar when there are no selected rows, and an actions toolbar when some rows are selected. ' +
-                'If you want to manually create the Select All UX in the table toolbar (or anywhere else), you can set `showSelectAll` prop to `false` so it will not be rendered in the table header.',
+                'If you want to manually create the bulk selection ("Select All") UX in the table toolbar (or anywhere else), set `hideBulkSelectionCheckbox` prop and use `<Table.BulkSelectionCheckbox>` component to add bulk selection in another location (under `<Table>`) - see `Table with custom bulk selection checkbox` example.',
               source: TableToolbarExampleRaw,
               compact: true,
             },
+            {
+              title: 'Table with Custom Bulk Selection Checkbox',
+              description:
+                'This is an example of having a custom bulk selection checkbox in the table toolbar using the `<Table.BulkSelectionCheckbox>` component',
+              source: TableBulkSelectionCheckboxExampleRaw,
+            },
```

Comment:
Why not making this example `compact`?

```suggestion
              compact: true,
            },
```

Reply:
👍 Done